canonical-web-and-design/ubuntu.com
# Switch Django to Flask

All the content has been reviewed going in, but this is so we can thoroughly QA it.

Fixes https://github.com/canonical-web-and-design/base-squad/issues/686

QA
--

Go through the whole site, check it against live. Especially check the complex bits - /download, /search, /blog, /kubernetes/docs, /legal

**NB: Feel free to QA using the demo link, but note that `/search` won't work on the demo. You'll see a 500 error, which is expected.**

*QAing locally:*

Grab the custom search creds from https://console.cloud.google.com/apis/credentials?project=ubuntu-search-1530889417216

`./run --env SEARCH_API_KEY={credential}`
